installer/resources/pacbot_app/files/DB.sql

```diff
@@ -752,6 +752,8 @@ CREATE TABLE IF NOT EXISTS `oauth_user_role_mapping` (
   `modifiedDate` datetime DEFAULT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+ALTER TABLE oauth_user_role_mapping ADD PRIMARY KEY (userRoleId);
```

Comment:
@johnrex-j 
Before adding primary key, you should delete all the duplicate values because in all previous deployments, there are duplicate data get loaded into the table because of not having a primary key/unique key and this will make problem to the already installed pacbotstack.
See here, https://github.com/tmobile/pacbot/blob/beaf950db0dbdd703f8dbc275d619c8e14a4f694/installer/resources/pacbot_app/files/DB.sql#L1039
During each deployemnt when this line get executed duplicate entries are created